index.html

```diff
@@ -337,8 +337,11 @@ <h3>
           |type:WakeLockType|:
         </p>
         <ol class="algorithm">
+          <li>Let |global| be <a>this</a>' <a>relevant global object</a>.
+          </li>
           <li>Let |document:Document| be the [=environment settings object /
-          responsible document=] of the <a>current settings object</a>.
+          responsible document=] of <a>this</a>' <a>relevant settings
+          object</a>.
```

Comment:
Domenic just used "<li>Let |document:Document| be the [=current settings object=]'s [=associated Document=]."

Reply:
I think it makes sense to clean this up in #307 rather than here, as it's part of the effort to stop mentioning "responsible document" altogether.